app/views/planets/show.html.erb

```diff
@@ -0,0 +1,52 @@
+<div class="banner banner-show" style="background-image: linear-gradient(-225deg, rgba(0,101,168,0.6) 0%, rgba(0,36,61,0.6) 50%), url('https://unsplash.com/photos/zpUsOQzByFg/download?force=true/download?force=true');">
```

Comment:
Please use the photo column in table planets

Reply:
done :-)